### PR TITLE
Implement header shell (no logout or game search)

### PR DESCRIPTION
**Pull request by Nathan-Laney**
NOTE: changed home() to index() in app.py for consistency with html filenames
page_index and current_page are vestigial for future highlighting functionality, not yet implemented

### CODE REVIEW

File: templates/_layout.html [hunk 1]
```diff
@@ -1,28 +1,39 @@
...
-    <meta name="viewport" content="width=device-width", initial-scale="1.0">
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/css/bootstrap.min.css"
+        integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link rel="stylesheet" href="/static/styles.css">
+    <title>Hello, world!</title>
```
Comment (tylerminnis) [inline templates/_layout.html:13]: change the title

File: templates/_layout.html [hunk 2]
```diff
@@ -1,28 +1,39 @@
...
     <title>My Site</title>
```
Comment (tylerminnis) [inline templates/_layout.html:14]: remove

File: templates/header.html [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+            <!-- ABOUT -->
+            <li class="nav-item">
+                <a class="nav-link" href="about.html">About Us</a>
+            </li>
+
+            <!-- ALL GAMES -->
+            <li class="nav-item">
+                <a class="nav-link" href="all_games.html">All Games</a>
+            </li>
+
```
Comment (tylerminnis) [inline templates/header.html:24]: clean up the whitespace possibly

**Review by tylerminnis — APPROVED** [review]
check comments out but overall LGTM

**Review by sumiv — APPROVED** [review]
let's get this merge !

**Pull request merged**